server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -1118,10 +1124,12 @@
   "dc/2wgh04kx6es88": "Median Income for Population Working in the Health Care and Social Assistance Industry",
   "dc/34t2kjrwbjd31": "Population Working in the Petroleum and Coal Products Manufacturing Industry",
   "dc/3ex11eg2q9hp6": "Median Income of Manufacturing Workers",
+  "dc/3jr0p7yjw06z9": "BLS Gambling Establishments",
```

Comment:
Just Gambling Establishments?

Reply:
Removed USC and changed names/descriptions for BLS SV